BrowserKit/Sources/Common/Extensions/URLExtension.swift

```diff
@@ -12,7 +12,6 @@ extension URL {
         self.init(string: string)
     }
 
-
```

Comment:
Sorry, silly me I didn't realize there were additional lint issues for last two commits. Hopefully BR passes 🤞🏼